### ISSUE TITLE
[TensorRT] ERROR: Could not register plugin creator:  FlattenConcat_TRT in namespace:  INFO:libs.detectors.jetson.mobilenet_ssd_v2:allocated buffers

### ISSUE THREAD

**Issue by nabil-bukhari** [neuralet/smart-social-distancing#42]
After I made fresh installation of Jetpack 4.3, I have build the docker files but I am facing the following issue on Jetson Nano device:

[TensorRT] ERROR: Could not register plugin creator:  FlattenConcat_TRT in namespace: 
INFO:libs.detectors.jetson.mobilenet_ssd_v2:allocated buffers

I did not make any changes to the config file and I am running the following commands:
sudo docker run -it --runtime nvidia --privileged -p 8080:8000 -v "$PWD/data":/repo/data neuralet/smart-social-distancing:latest-jetson-nano

![Screenshot from 2020-07-29 19-12-35](https://user-images.githubusercontent.com/68958543/88828300-bbf63180-d1d3-11ea-9f53-15f9d672e44b.png)

**Comment by xYoshiox** [comment]
I have the same problem after upgrading the repo

**Comment by nabil-bukhari** [comment]
Did you find the solution? as I am still struggling

**Comment by tychotdeback** [comment]
I have the exact same issue on my jetson nano using jetpack 4.3. A solution would be very welcome!

**Comment by renzodgc** [comment]
Hey, we just got around checking these old issues, sorry about the delay.

I don't believe we ever had an issue with Jetpack 4.3, although on some versions we've experienced issues with the jetson nano image (that were fixed), it never happened to us that the processor started processing frames and stopped (as far as I can remember)

I presume this issue has been fixed. Feel free to try to reproduce it if you are still interested and report back If the issue remains. Elsewise I'll close this issue in a week :)
